rpc/ub/plot.sh

```diff
@@ -128,19 +128,19 @@ gen_csv() {
     local VAR_OPT="$1"; shift
     local VALUES="$*"
 
-    local NR_MSGS=$(get_val nr_msgs $COMMON_OPTS)
+    local NR_MSGS=$(get_val nr_msgs "$COMMON_OPTS")
 
     echo "# $VAR_OPT time msg/s MB/s" >"$OUT"
 
     for x in $VALUES; do
 	local OPTS="${VAR_OPT}=${x},${COMMON_OPTS}"
 	echo "----------[ $OPTS ]----------"
 
-	rpc-ub -o $OPTS | tee "$TMP"
-	[ ${PIPESTATUS[0]} -eq 0 ] || exit ${PIPESTATUS[0]}
+	rpc-ub -o "$OPTS" | tee "$TMP"
+	[ "${PIPESTATUS[0]}" -eq 0 ] || exit "${PIPESTATUS[0]}"
 
-	local NR_CONNS=$(get_val nr_conns $OPTS)
-	local MSG_LEN=$(get_val msg_len $OPTS)
+	local NR_CONNS=$(get_val nr_conns "$OPTS")
+	local MSG_LEN=$(get_val msg_len "$OPTS")
```

Comment:
@zohebkhann  Are these "OPTS" space separated options? If yes, then you can't put them in double quotes as they will be passed as single argument.

Reply:
Changed as per comment.